src/GameplayHud.cpp

```diff
@@ -198,31 +198,53 @@ void GameplayHud::load() {
 }
 
 void GameplayHud::render() const {
+
     if (countdown_value_ > 0) {
         locked_scoreboard_.render(glm::mat4());
     } else {
-        scoreboard_.render(glm::mat4());
+        auto transform = (num_ai_ == 3) ? glm::translate(glm::mat4(1.f), glm::vec3(0.7f, -.6f, 0.f)) : glm::mat4();
+        scoreboard_.render(transform);
     }
 
     if (countdown_value_ > 0) {
         countdown_.render(glm::mat4());
     }
 
     for (auto& score : scores_) {
-        score.render(glm::mat4());
+
+        if (num_ai_ == 3) {
+            auto transform = glm::translate(glm::mat4(1.f), glm::vec3(0.7f, -.6f, 0.f));
+            score.render(transform);
+        } else {
+            score.render(glm::mat4());
+        }
     }
 
     for (unsigned int i = 0; i < it_pointers_.size(); i++) {
         it_pointers_[i].render(it_pointer_transforms_[i]);
+
+        //If singleplayer, only render the first pointer
+        if (num_ai_ == 3) {
+            break;
+        }
     }
 
     for (auto& holder : powerup_holders_) {
         holder.render(glm::mat4());
```

Comment:
The holder is really tiny for single player, we should make it bigger on singe player.